doc/pset.mediawiki

```diff
@@ -21,6 +20,9 @@ signatures for an input while the input does not have a complete set of signatur
 The signer can be offline as all necessary information will be provided in the
 transaction.
 
+This is a modification of BIP 174 for the Elements sidechain. It includes changes necessary
+for Confidential Transactions, Confidential Assets, and Peg-in transactions.
+
```

Comment:
Commit 0b3b9edcfe5bcf74c1239b073e00a1ea3f788682 is just the PSET changes.

Reply:
The people who will be implementing support for PSET in their code have to use the full document, they cannot use commit as the reference. I concur with @jgriffiths that the doc should be an addendum rather than edit. I think that the described problems (need to read diff PSET/PSBT instead of the document, need to sync with PSBT as it is updated) are significant. It is the additions to the standard that are proprietary, not the standard itself. Therefore the additions should be described in the spec.

Reply:
*proprietary in a sense that they use the `PSBT_PROPRIETARY ` type designed for proprietary extensions, not that the changes are somehow not open.